THIRD_PARTY_NOTICES.md

```diff
@@ -953,7 +953,7 @@ Apache License
 
 ### @newrelic/security-agent
 
-This product includes source derived from [@newrelic/security-agent](https://github.com/newrelic/csec-node-agent) ([v0.0.4](https://github.com/newrelic/csec-node-agent/tree/v0.0.4)), distributed under the [Apache-2.0 License](https://github.com/newrelic/csec-node-agent/blob/v0.0.4/LICENSE):
+This product includes source derived from [@newrelic/security-agent](https://github.com/newrelic/csec-node-agent) ([v1.0.0](https://github.com/newrelic/csec-node-agent/tree/v1.0.0)), distributed under the [UNKNOWN License](https://github.com/newrelic/csec-node-agent/blob/v1.0.0/LICENSE):
```

Comment:
Version 1.0, public release, but still has the pre-release license instead of Apache? Is that correct?

Reply:
Yes, security agent is not under apache 2.0 license.

Reply:
Feels very premature to release a 1.0.0 of software that isn't GA nor has any customers. If you hit issues during the beta phase and need to release a breaking change it'd be 2.0.0. When you're in the 0.x line it's still ok to release breaking changes.

Reply:
Agree. In that case we have to unpublish 1.0.0 from NPM.